lib/api_client/execute_request.js

```diff
@@ -6,11 +6,62 @@ const Q = require('q');
 const url = require('url');
 const utils = require("../utils");
 const ensureOption = require('../utils/ensureOption').defaults(config());
+const request = require('request');
```

Comment:
It's the only npm package for doing http requests that still supports node < 10. There are no additional security vulnerabilities after I added it. And HTTP as a protocol is not changing since years. It's not ideal but it gets the job done.